REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.4.0
-Compat 0.8.5
+Compat 0.10.0
```

Comment:
too conservative - 0.9.5 introduced `take!`, and you're not using transcode or `.=`